src/pages/index.js

```diff
@@ -105,80 +200,45 @@ function handleAddCardSubmit(inputValues) {
     });
 }
 
-// Function to handle profile edit
 function handleEditProfileSubmit(inputValues) {
-  userInfo.setUserInfo({
-    name: inputValues.title,
-    description: inputValues.description,
-  });
-  editProfilePopup.close();
+  console.log("Edit Profile Input Values", inputValues);
+  api
+    .editProfile(inputValues.title, inputValues.description)
+    .then(() => {
+      userInfo.setUserInfo({
+        name: inputValues.title,
+        description: inputValues.description,
+      });
+      editProfilePopup.close();
+    })
+    .catch((err) => {
+      console.error("Error updating profile:", err);
+    })
+    .finally(() => {
+      editProfilePopup.setLoading(false);
+    });
 }
 
-// Function to create a card
-function createCard(data) {
-  const card = new Card(
-    data,
-    "#card-template",
-    handleImageClick,
-    handleDeleteCard
-  );
-  return card.getView();
+function handleAvatarSubmit(inputValues) {
+  profileAvatarPopup.setLoading(true);
+  api
+    .updateAvatar(inputValues.avatar)
+    .then(() => {
+      userInfo.updateAvatar(inputValues.avatar);
+      profileAvatarPopup.close();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      profileAvatarPopup.setLoading(false);
+    });
 }
 
-// Function to render cards
 function renderCards(cardData) {
   const cardElement = createCard(cardData);
   section.addItem(cardElement);
   addCardPopup.close();
   addForm.reset();
   addCardFormValidator.disableButton();
```

Comment:
`renderCards` should not close, reset and disable because it's just for rendering one card 

You do all that in lines 194-196